src/main/components/RequestBar.jsx

```diff
@@ -1,26 +1,24 @@
 import React, { useState } from 'react';
 
-const UrlBar = (props) => {
-  const method = (props.AorB === 'B' ? 'POST' : 'GET')
+const RequestBar = (props) => {
+  const method = (props.SourceOrDest === 'dest' ? 'POST' : 'GET')
   const [selected, setSelected] = useState(method);
 
-  function sendFetch () {
+  function sendFetch (e) {
+    e.preventDefault();
     const sendingObj = { method: selected, mode: 'cors' };
     if (props.SourceOrDest === 'dest') sendingObj.body = JSON.stringify(props.requestBody);
 
-    // console.log('hello;', document.querySelector('#urlInput').value);
-
     fetch(document.querySelector('#urlInput').value, sendingObj)
```

Comment:
This maybe because you were testing, however It would be better to read the request url from state instead of getting it from the DOM. There is a slight performance improvement from reading the fetch url from state vs telling react to reach out to the DOM to get the value of the input URL.  We can discuss this more if needed